src/MorseL/Extensions/MorseLOptions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Extensions.Options;
+using MorseL.Scaleout;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MorseL.Extensions
```

Comment:
This mirrors SignalR's namespace for their AppBuilder/IServiceProvider extensions. I do agree that MorseLOptions inside of MorseL.Extensions is strange. Perhaps ServiceExtensions?